### DISCUSSION TITLE
Audios里，有时Transcription与右侧Note对应有bug

### DISCUSSION THREAD

**Discussion by JettZou99** [ZuodaoTech/everyone-can-use-english#700] [General]
截图如下：
![image](https://github.com/xiaolai/everyone-can-use-english/assets/6868534/c67c777e-1385-499f-8878-e74e89de9676)
语音文件如下：
[synthesis.zip](https://github.com/user-attachments/files/15922251/synthesis.zip)
谢谢项目开源共享，祝越来越好！

**Comment by an-lee** [comment]
感谢反馈。我测试一下看。
